bsip-0083.md

```diff
@@ -0,0 +1,74 @@
+```
+BSIP: 0083
+Title: Decouple BitAssets from Platform Governance Process
+Authors:    
+   Alfredo Garcia
+   Christopher Sanborn
+   John M. Jones
+   Michel Santos
+   Nathan Hourt
+   Peter Conrad
+   Ryan R. Fox
+Status: Draft
+Type: Protocol
+Obsoletes: BSIP-0016, BSIP-0058, BSIP-0059, BSIP-0075, BSIP-0076
+Created: 2019-10-07
+Discussion: https://github.com/bitshares/bsips/issues/239
+Worker: <TBD>
+```
+
+# Abstract
+This BSIP separates the governance of the technical security of the blockchain from the governance of BitAssets.  BitAssets are a collection of special case SmartCoin which conflate blockchain governance and market effectiveness. To best serve the BitShares protocol, the existing set of BitAssets must be reconfigured to separate the concerns of block production, price feeds, and market operations; that is to say: deterministic consensus, non-deterministic external data, and smart contract settings.
+
+Therefore, this BSIP defines a process for BitAssets management to ensure protocol security remains the singular focus of the block producer role, and give BitAssets the best chance for long-term viability by nominating a new custodian to quickly react to market conditions.  Furthermore, the portion of BTS collateral below the full valuation of a debt asset, as defined by its price feed, that is borrowed from the blockchain, shall not contribute to the voting power of the debtor.
+
+# Motivation
+Block production is an essential component of the BitShares platform itself, and as such should be expected to take an agnostic and neutral view towards the products deployed within the platform. Therefore, this BSIP will define a process for BitAsset management to ensure protocol security remains the singular focus of the block producer role and give BitAssets the best chance for long-term stability by nominating a new custodian to quickly react to market conditions. In order for the distributed consensus mechanism of DPOS to achieve adequate decentralization, it is imperative that block producers maintain strict independence from the politics of individual products deployed on the platform. 
+
+Governance is an important component of the core BTS token and effective application of governance throughout the BitShares protocol is imperative to the long-term success of the project.  A number of BSIPs have addressed the shortcomings of the existing vote calculation mechanism. This BSIP changes vote calculation which currently confers governance rights which are intended for assignment to the protocol.
```

Comment:
I really like the idea of removing the 1x CR from active vote weight & allowing the remaining surplus BTS as debtor vote weight, but I think it'd be best in its own BSIP since this would require a hardfork change where as the other requested changes do not and have fairly different focuses.

Reply:
Removed.